packages/puppeteer-crawler/src/internals/stealth/index.ts

```diff
@@ -1,10 +1,12 @@
 import { Page, Browser } from 'puppeteer';
 import { cryptoRandomObjectId } from '@apify/utilities';
-import globalLog from '../utils_log';
+import {
+    logUtils,
+    keys,
+} from '@crawlers/core';
 import * as hidingTricks from './hiding_tricks';
-import { keys } from '../typedefs';
 
-const log = globalLog.child({ prefix: 'Stealth' });
+const log = logUtils.child({ prefix: 'Puppeteer Stealth' });
```

Comment:
lets keep the prefix
```suggestion
const log = logUtils.child({ prefix: 'Stealth' });
```

Reply:
I mean right now it only applies to puppeteer, and if you use both puppeteer and playwright crawlers..you could think stealth applies to both. I will remove it, but just making sure

Reply:
but this will never log anything for playwright, or will it? its in the puppeteer package

Reply:
It won't, you're right, but shouldn't we implement it for Playwright too?

Reply:
the plan is actually to remove it in v3 in favour or fingerprints

Reply:
Oh. Well then 😁
I'll apply the change then 👍